src/braft/node.cpp

```diff
@@ -123,6 +123,7 @@ inline int random_timeout(int timeout_ms) {
 }
 
 DEFINE_int32(raft_election_heartbeat_factor, 10, "raft election:heartbeat timeout factor");
+BRPC_VALIDATE_GFLAG(raft_election_heartbeat_factor, brpc::PositiveInteger);
 static inline int heartbeat_timeout(int election_timeout) {
```

Comment:
The flags configured with BRPC_VALIDATE_GFLAG allow users to change them at runtime, but raft_election_heartbeat_factor don't match the requirement. So, do the defence in the function |heartbeat_timeout| should be better?

Reply:
It's meaningless for raft_election_heartbeat_factor less than 1.

Reply:
Add raft_election_heartbeat_factor defence in heartbeat_timeout.
If you think this change is not necessary, please feel free to close it.